lib/cea/sei_processor.js

```diff
@@ -0,0 +1,122 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.cea.SeiProcessor');
+
+
+/**
+ * H.264 SEI NALU Parser used for extracting 708 closed caption packets.
+ */
+shaka.cea.SeiProcessor = class {
+  constructor() {
+    /**
+     * Length of the buffered SEI data.
+     * @private {number}
+     */
+    this.seiLength_ = 0;
+
+    /**
+     * Buffer for the supplemental enhancement information (SEI) network
+     * abstraction layer unit (NALU).
+     * @private @const {!Uint8Array}
+     */
+    this.seiArray_ = new Uint8Array(5 * 1024);
+  }
+
+
+  /**
+   * Clears processor state.
+   */
+  clear() { this.seiLength_ = 0; }
+
+
+  /**
+   * Returns the length of SEI data pending to be processed.
+   * @return {number} Length of SEI data pending to be processed.
+   */
+  getLength() { return this.seiLength_; }
```

Comment:
It looks like this isn't used anywhere.  What is it for?

Reply:
This was implemented while keeping a potential TS Parser in mind. Although yes, these are currently unused. I can remove them for now.